system-parachains/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -244,3 +253,119 @@ impl pallet_salary::Config<FellowshipSalaryInstance> for Runtime {
 	// Total monthly salary budget.
 	type Budget = ConstU128<{ 100_000 * USDT_UNITS }>;
 }
+
+parameter_types! {
+	// TODO: reference the constant value from common crate with polkadot-sdk 1.5
+	pub const FellowshipTreasuryPalletId: PalletId = PalletId(*b"py/feltr");
+	pub const ProposalBond: Permill = Permill::from_percent(1);
+	pub const ProposalBondMinimum: Balance = 5 * DOLLARS;
+	pub const ProposalBondMaximum: Balance = 10 * DOLLARS;
+	pub const SpendPeriod: BlockNumber = 7 * DAYS;
+	pub const Burn: Permill = Permill::from_percent(0);
+	pub const PayoutSpendPeriod: BlockNumber = 30 * DAYS;
+	// The asset's interior location for the paying account. This is the Fellowship Treasury
+	// pallet instance (which sits at index 65).
+	pub FellowshipTreasuryInteriorLocation: InteriorMultiLocation = PalletInstance(65).into();
+}
+
+/// [`PayOverXcm`] setup to pay the Fellowship Treasury.
+pub type FellowshipTreasuryPaymaster = PayOverXcm<
+	FellowshipTreasuryInteriorLocation,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	VersionedMultiLocation,
+	VersionedLocatableAsset,
+	LocatableAssetConverter,
+	VersionedMultiLocationConverter,
+>;
+
+pub type FellowshipTreasuryInstance = pallet_treasury::Instance1;
+
+impl pallet_treasury::Config<FellowshipTreasuryInstance> for Runtime {
+	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
+	type PalletId = FellowshipTreasuryPalletId;
+	type Currency = Balances;
+	// This parameter guards a deprecated call and should not be used.
+	// TODO: replace by NeverEnsure with polkadot-sdk 1.5
+	type ApproveOrigin = EnsureRoot<AccountId>;
+	type RejectOrigin = EitherOfDiverse<
+		EnsureRoot<AccountId>,
+		EitherOfDiverse<EnsureXcm<IsVoiceOfBody<GovernanceLocation, TreasurerBodyId>>, Fellows>,
+	>;
+	type RuntimeEvent = RuntimeEvent;
+	// This type should never be triggered since it meant for deprecated functionality.
+	type OnSlash = ();
+	type ProposalBond = ProposalBond;
+	type ProposalBondMinimum = ProposalBondMinimum;
+	type ProposalBondMaximum = ProposalBondMaximum;
+	type SpendPeriod = SpendPeriod;
+	type Burn = Burn;
+	type BurnDestination = ();
+	type SpendFunds = ();
+	type MaxApprovals = ConstU32<100>;
+	type SpendOrigin = EitherOf<
```

Comment:
We should have `EitherOf3`, `EitherOf4` etc to make those code bit easier to read. But that can happen in future.

Reply:
"Either" refers to a choice of _two_ possibilities, so that naming doesn't make sense. But `AtLeastOneOf` or `AnyOf` would do.